cmd/manager/main.go

```diff
@@ -118,6 +118,21 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Define a kubeClient for any processes that need to run during operator startup or independent routines to use
```

Comment:
From: https://github.com/openshift/aws-account-operator/pull/668#discussion_r810034883
```suggestion
	// Define a kubeClient for any processes that need to run during operator startup or independent routines to use
	// We should avoid using this kubeClient except for when necessary and utilize the operator-sdk provided client as much as possible.
	// The operator-sdk kube client provides a level of caching that we don't get with building our own this way.
```